src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Nodes/NodePropertyFilter.cs

```diff
@@ -55,52 +55,40 @@ public bool CanApplyFilter(Type type, ValidForFlag validForFlag)
         }
 
         /// <summary>
-        /// Creates a string from the filter property and values to be used in the Urn to query the SQL objects
-        /// Example of the output:[@ IsSystemObject = 0]
+        /// Creates a string representation of a given node filter, which is combined in the INodeFilter interface to construct the filter used in the URN to query the SQL objects.
+        /// Example of the output: (@ IsSystemObject = 0)
         /// </summary>
         /// <returns></returns>
-        public string ToPropertyFilterString()
+        public string ToPropertyFilterString(Type type, ValidForFlag validForFlag)
```

Comment:
we already have similar method above, right?

Reply:
Where?

Reply:
https://github.com/microsoft/sqltoolsservice/pull/1608/files#diff-3411bb2896907eccf0500c3b3c71db52d3fb27d69497c5cceb8cb249b9a88bc2R36